### DISCUSSION TITLE
Email channel does not work

### DISCUSSION THREAD

**Discussion by ilyakonovalenko** [chatwoot/chatwoot#3145] [Self hosted] [answered]
Hello,

I configured postfix to pipe emails to Chatwoot with:
`cd /home/chatwoot/chatwoot && bin/rails action_mailbox:ingress:postfix URL="https://DOMAIN/rails/conductor/action_mailbox/inbound_emails" INGRESS_PASSWORD="PASSWORD"`

However, Postfix responds with `status=deferred (HTTP 422)` in the logs. 

Do you guys have any idea?

**Comment by sojan-official** [comment, marked as answer]
have you configured the environment variable for postfix? 
```
RAILS_INBOUND_EMAIL_SERVICE=relay
```
ref: https://www.chatwoot.com/docs/self-hosted/configuration/features/email-channel/conversation-continuity#configuring-inbound-reply-emails
